src/Microsoft.OData.Core/ODataInputContext.cs

```diff
@@ -272,26 +272,26 @@ public virtual Task<ODataCollectionReader> CreateCollectionReaderAsync(IEdmTypeR
 #endif
 
         /// <summary>
-        /// Read the property from the input and
+        /// Read the EDM structural property from the input and
         /// return an <see cref="ODataProperty"/> representing the read property.
         /// </summary>
-        /// <param name="property">The <see cref="IEdmProperty"/> producing the property to be read.</param>
+        /// <param name="edmStructuralProperty">The <see cref="IEdmProperty"/> producing the property to be read.</param>

```

Comment:
yes, that was causing a warning "CA1716:IdentifiersShouldNotMatchKeywords" which was suppressed.